hikes.properties

```diff
@@ -0,0 +1,5 @@
+# this is hikes.properties
+vernal falls
+yosemite falls
+4 mile
```

Comment:
is the 4 mile trail the same trail as the glacier pt trail?
![git-logo](https://cloud.githubusercontent.com/assets/18578473/14689433/222a62fc-06fc-11e6-8741-49e904e01cdd.png)